chatwoot/chatwoot
# fix: Disable showing read messages in unread view

Fixes #4114 

**How to test**

1. Create  a conversation from the widget
2. Give a reply from the dashboard
3. Read the message in widget
4. Add the extension [url-throttler ](https://chrome.google.com/webstore/detail/url-throttler/kpkeghonflnkockcnaegmphgdldfnden) and configure a delay for conversations end point 
`https://app.chatwoot.com/api/v1/widget/conversations*`
5. Reload the widget page and check read messages are appearing or not. Before this change, the read messages were appearing in unread view
